lib/route.js

```diff
@@ -322,7 +320,7 @@ function buildRouting (options) {
   }
 
   // HTTP request entry point, the routing has already been executed
-  function routeHandler (req, res, params, context) {
+  function routeHandler (req, res, params, context, query) {
```

Comment:
Do we have a test that shows using `qs` over `querystring`?

Reply:
There is a [custom-querystring-parser.test.js](https://github.com/fastify/fastify/blob/main/test/custom-querystring-parser.test.js) that tests the custom parser option. You can use any querystring parser that you want. I don't see the need to test particular implementations.